src/WebStarter/appsettings.json

```diff
@@ -164,6 +164,20 @@
           "CompletionCost": 0.002
         }
       ]
+    },
+    {
+      "Provider": "google-ai",
+      "Models": [
+        {
+          "Name": "gemini-2.0-flash-exp",
+          "ApiKey": "",
+          "Type": "chat",
+          "MultiModal": true,
+          "RealTime": true,
```

Comment:
Already removed in the new version "RealTime", Please use `"Type": "realtime"` instead.